src/store/appType/whatsapp_cloud/getters.js

```diff
@@ -0,0 +1,23 @@
+export default {
+  wabaId(state) {
```

Comment:
A good tip is use `mapState` to get this type of info. Usually we use getters to add some treatment 

https://vuex.vuejs.org/guide/state.html#the-mapstate-helper